arm-templates/ExampleWithDiagnosticDettings/README.md

```diff
@@ -0,0 +1,12 @@
+# Azure Database for PostgreSQL with CanNotDelete Lock
```

Comment:
typo in the folder name: "ExampleWithDiagnosticDettings" should be 'ExampleWithDiagnostic**S**ettings'